From b8b8c822bbb0e1c16f49f4c22a7195bf0cde6331 Mon Sep 17 00:00:00 2001
From: hmtheboy154 <buingoc67@gmail.com>
Date: Wed, 4 Jun 2025 04:55:20 +0000
Subject: [PATCH 42/42] treble.mk: Yea no I'm not gonna fix the mess in
 compatibility matrix level 8

Which mean go back to software gatekeeper@1.0 HIDL

Signed-off-by: hmtheboy154 <buingoc67@gmail.com>
---
 manifest.xml     | 2 +-
 manifest_omx.xml | 2 +-
 treble.mk        | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/manifest.xml b/manifest.xml
index 00bce86..556595e 100644
--- a/manifest.xml
+++ b/manifest.xml
@@ -1,4 +1,4 @@
-<manifest version="8.0" type="device" target-level="8">
+<manifest version="8.0" type="device" target-level="7">
     <hal format="hidl">
         <name>android.hardware.audio</name>
         <transport>hwbinder</transport>
diff --git a/manifest_omx.xml b/manifest_omx.xml
index 6715bb3..a771d0b 100644
--- a/manifest_omx.xml
+++ b/manifest_omx.xml
@@ -1,4 +1,4 @@
-<manifest version="8.0" type="device" target-level="8">
+<manifest version="8.0" type="device" target-level="7">
     <hal format="hidl">
         <name>android.hardware.media.omx</name>
         <transport>hwbinder</transport>
diff --git a/treble.mk b/treble.mk
index fdfa95b..9c08a4a 100644
--- a/treble.mk
+++ b/treble.mk
@@ -42,7 +42,7 @@ PRODUCT_PACKAGES += \
 
 # Gatekeeper HAL
 PRODUCT_PACKAGES += \
-    com.android.hardware.gatekeeper.nonsecure
+    android.hardware.gatekeeper@1.0-service.software
 
 # Health HAL
 PRODUCT_PACKAGES += \
-- 
2.43.0

